Makefile.common

```diff
@@ -118,7 +118,7 @@ endif
 %: common-% ;
 
 .PHONY: common-all
-common-all: precheck style check_license lint unused build test
+common-all: precheck restricted_imports style check_license lint unused build test
```

Comment:
I am still missing a lot of context, so I am not completely sure that this is the best place to put this, suggestions definitely welcome.